app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueModal.scss

```diff
@@ -0,0 +1,84 @@
+//.message {
+//  max-width: 600px;
+//  b {
+//    font-family: $FONT-SEMIBOLD;
+//  }
+//}
+
+.add-issue-id-title {
+  margin: 0;
+  padding: 10px;
+  color: $COLOR--gray-47;
+  background-color: $COLOR--white-four;
+  font-size: 12px;
+  font-family: $FONT-SEMIBOLD;
+  text-transform: uppercase;
+}
+
+.link-issue-form-wrapper {
+  padding: 17px 9px 0 12px;
+  margin-bottom: 10px;
+  box-sizing: border-box;
+}
+
+.multiple-systems-title {
+  display: inline;
+  min-width: 140px;
+  padding: 0;
+  font-size: 13px;
+  font-family: $FONT-REGULAR;
+  text-align: center;
+}
+
+.dropdown-container {
+  display: inline-block;
+  width: 350px;
+  padding: 0;
+  margin-left: 20px;
+  margin-right: 25px;
+}
+
+.none-description {
+  display: none;
+}
+
+.issue-fields-list-item {
+  position: relative;
+  margin-bottom: 18px;
+}
+
+.inputs-group-block {
+  margin-bottom: 10px;
+}
+
+.remove-issue-fields-button {
+  position: absolute;
+  right: -4px;
+  top: 25px;
+  display: block;
+  width: 17px;
+  height: 17px;
+  vertical-align: middle;
+  transition: transform 200ms ease-in-out;
+  cursor: pointer;
+
+  svg {
+    fill: $COLOR--gray-60;
+  }
+  &:hover {
+    transform: scale(1.2);
+
+    svg {
+      fill: $COLOR--gray-47;
+    }
+  }
+}
+
+.add-issue-button {
+  margin-left: 160px;
+}
+
+.add-issue-button-icon {
+  font-size: 21px;
+  margin: 0 3px 0 0 !important;
```

Comment:
FYI. Should be a really a strong reason for using !important

Reply:
Fixed